streams-activitypub-graph/src/test/scala/org/apache/streams/activitypub/graph/test/cases/GraphDatabaseServerAvailableTest.scala

```diff
@@ -0,0 +1,40 @@
+package org.apache.streams.activitypub.graph.test.cases
+
+import org.apache.http.HttpStatus
+import org.apache.streams.activitypub.graph.test.ActivityPubGraphTestSuiteExtension
+import org.awaitility.Awaitility.await
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Order
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.slf4j.LoggerFactory
+
+import scala.concurrent.duration.MINUTES
+import scala.concurrent.duration.SECONDS
+import scala.util.Try;
+
+/**
+ * ActivityPubServletsTestSuite
+ */
+@ExtendWith(Array(classOf[ActivityPubGraphTestSuiteExtension]))
```

Comment:
the ExtendWith annotation(s) on test class in conjuction with the ParameterResolver implementation of the extension(s), is what supplies the 'helper'(s) to test methods that need it/them